docs/source/formats.rst

```diff
@@ -63,8 +59,8 @@ There are some common parameters across data set types:
   The name of the data set, for display purposes. Only used in the GUI.
 `scan_size`
   In the GUI, we generally support visualizing data containing rectangular 2D scans. For
-  some data set types, you can specify a scan_size as a tuple `(y, x)`; for
-  others it is even required. When using the Python API, you are free to use n-dimensional
+  some data set types, you can specify a scan_size as (scan width | scan height); for others
+  it is not even required. When using the Python API, you are free to use n-dimensional
   `scan_size`, if the data set and chosen analysis supports it.
```

Comment:
Thanks for keeping the documentation in mind!

`(y, x)` is `(height, width)`. This is also what is described in the "we follow numpy convention" paragraph - I think we should keep that!

The "for others it is even required" was referencing formats like raw or seq that absolutely need a scan size argument to properly read the data. Maybe this is misleading and that part of the sentence can be completely removed.